website/docusaurus.config.ts

```diff
@@ -441,6 +444,8 @@ export default async function createConfigAsync() {
           blog: {
             // routeBasePath: '/',
             path: 'blog',
+            sortPosts: (a: BlogPost, b: BlogPost): number =>
+              b.metadata.date.getTime() - a.metadata.date.getTime(),
```

Comment:
Test this by applying this to the dogfood blog plugin.

For example you can write a blog post with an old date, and make it appear in 1st position thanks to a front matter field?